addOns/scripts/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 ### Added
+- 'Copy as curl command menu.js' added change that prevents curl from adding User-Agent when no user Agent should be present
```

Comment:
- IMO it would be better if placed under the Changed section since it's changing an existing script.
 - Add dot at the end.
 - `user Agent` → `User-Agent`